doc/ref/states/requisites.rst

```diff
@@ -532,6 +533,26 @@ The ``runas`` global option is used to set the user which will be used to run th
 
 In the above state, the pip command run by ``cmd.run`` will be run by the daniel user.
 
+runas_passwd
+~~~~~~~~~~~~
+
+.. versionadded:: 2017.7.2
+
+The ``runas_passwd`` global option is used to set the password used by the runas
+global option. This is required by ``cmd.run`` on Windows when ``runas`` is
+specified. It will be set when ``password`` is defined in the state.
+
+.. code-block:: yaml
+
+    run_script:
+      cmd.run:
+        - name: Powershell -NonInteractive -ExecutionPolicy Bypass -File C:\\Temp\\script.ps1
+        - runas: frank
+        - password: supersecret
```

Comment:
This is in the state file. I think it stays password. That's what's documented. If we rename it, then we'll have to do a bunch of deprecation stuff.